src/cdk/layout/breakpoints-observer.ts

```diff
@@ -71,7 +71,7 @@ export class BreakpointObserver implements OnDestroy {
    * @param value One or more media queries to check.
    * @returns A stream of matches for the given queries.
    */
-  observe(value: string | string[]): Observable<BreakpointState> {
+  observe(value: string | readonly string[]): Observable<BreakpointState> {
```

Comment:
Shouldn't this be `ReadonlyArray<string>`?

Reply:
`ReadonlyArray<string>` and `readonly string[]` have exactly the same meaning. I can adjust if the `ReadonlyArray<>` style is more adopted in the repo.

PS: It would be nice to pick one of the styles and use it consistently. [In angular repo someone created an issue relating this]( https://github.com/angular/angular/issues/33135).

Reply:
I do prefer `readonly string[]` myself